go.sum

```diff
@@ -1,25 +1,19 @@
-4d63.com/gochecknoglobals v0.1.0/go.mod h1:wfdC5ZjKSPr7CybKEcgJhUOgeAQW1+7WcyK8OvUilfo=
```

Comment:
Did you update to the 1.18 locally? I have the same go.sum tidy changes, but I wasn't uploading them unless the whole team updates. And if the whole team updates, there is no sense in keeping our go.mod file in 1.17 and not 1.18. 

E.g, if someone git checks out with go 1.17 this updated go.sum, then he would need to do go tidy to be able to compile.

Reply:
do you even get these changes with 1.18 when doing `go mod tidy -compat=1.17`?

Reply:
yes. I actually executed `go mod tidy -compat=1.17`

Reply:
I will downgrade go and update pr then.

Reply:
Independent update fixed that